org.eclipse.jdt.ls.product/scripts/jdtls.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import glob
+import os
+import platform
+import re
+import subprocess
+import sys
+from pathlib import Path
+
+def get_java_executable(validate_java_version):
+	java_executable = 'java'
+
+	if 'JAVA_HOME' in os.environ:
+		java_exec_to_test = Path(os.environ['JAVA_HOME']) / 'bin' / 'java'
+		if java_exec_to_test.is_file():
+			java_executable = java_exec_to_test.resolve()
+
+	if not validate_java_version:
+		return java_executable
+
+	out = subprocess.check_output([java_executable, '-version'], stderr = subprocess.STDOUT, universal_newlines=True)
+
+	matches = re.finditer(r"(?P<major>\d+)\.\d+\.\d+", out)
+	for matchNum, match in enumerate(matches):
+		java_major_version = int(match.group("major"))
+
+		if java_major_version < 11:
+			raise Exception("jdtls requires at least Java 11")
+
+		return java_executable
+
+	raise Exception("Could not determine Java version")
+
+def find_equinox_launcher(jdtls_base_directory):
+	plugins_dir = jdtls_base_directory / "plugins"
+	launchers = glob.glob('org.eclipse.equinox.launcher_*.jar', root_dir = plugins_dir)
```

Comment:
```suggestion
	launchers = plugins_dir.glob('org.eclipse.equinox.launcher_*.jar')
```

This would also eliminate the glob import

Reply:
Needed to change the `if` to a `for` loop because `Path.glob` returns a generator object.